flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/api/batch/table/validation/CalcValidationTest.scala

```diff
@@ -102,14 +102,6 @@ class CalcValidationTest extends TableTestBase {
       case _: ValidationException => //ignore
     }
 
-    try {
-      util.addTable[(Int, Long, String)]("Table2")
-      .select('_1 as '*, '_2 as 'b, '_1 as 'c)
```

Comment:
Why did you remove this test? Is the field reference a problem here? Can we just change it to strings?

```
.select('_1 as "*", '_2 as "b", '_1 as "c")
```

Reply:
It is very unlikely case that is not worth to have a separate input type strategy for.